apis/Google.Cloud.Asset.V1/docs/history.md

```diff
@@ -1,5 +1,19 @@
 # Version history
 
+# Version 2.3.0, released 2020-08-25
+
+- [Commit 8ef5dc2](https://github.com/googleapis/google-cloud-dotnet/commit/8ef5dc2): feat: add AnalyzeIamPolicy and ExportIamPolicyAnalysis RPCs
+- [Commit e7b75e6](https://github.com/googleapis/google-cloud-dotnet/commit/e7b75e6):
```

Comment:
I suspect the doc updates aren't worth mentioning here - they tend to just happen anyway. I'd put the "feat" inline with the commit, and not have the list.

Reply:
Done